# HG changeset patch
# User Andreas Weber <andy.weber.aw@gmail.com>
# Date 1449953664 -3600
#      Sat Dec 12 21:54:24 2015 +0100
# Node ID 40da964c3add342fe31801b3b651629bde90b43d
# Parent  5357423bca0a3e5e985ba63338131d3db48bddda
time.cc: codesprint: Fix parenthesis from cset:7810a10772e0

diff --git a/libinterp/corefcn/time.cc b/libinterp/corefcn/time.cc
--- a/libinterp/corefcn/time.cc
+++ b/libinterp/corefcn/time.cc
@@ -483,17 +483,17 @@ you're absolutely sure the date string w
     print_usage ();
 
   std::string str = args(0).xstring_value ("strptime: argument STR must be a string");
 
   std::string fmt = args(1).xstring_value ("strptime: FMT must be a string");
 
   octave_strptime t (str, fmt);
 
-  return ovl (octave_value (mk_tm_map (t), t.characters_converted ()));
+  return ovl (octave_value (mk_tm_map (t)), t.characters_converted ());
 }
 
 /*
 %!test
 %! fmt = "%Y-%m-%d %H:%M:%S";
 %! s = strftime (fmt, localtime (time ()));
 %! ts = strptime  (s, fmt);
 %! assert (isstruct (ts));
